app/jobs/sync/bitflyer/update_missing_deposit_job.rb

```diff
@@ -0,0 +1,22 @@
+class Sync::Bitflyer::UpdateMissingDepositJob
+  include Sidekiq::Worker
+  sidekiq_options queue: :default, retry: true
+
+  def initialize(channel_id)
+    @channel = Channel.find(channel_id)
+  end
+
+  def perform
+    return if @channel.deposit_id.present?
```

Comment:
Can we put the core of this code in a service class? It would be nice to separate the _what_ gets run from the _where_ and _when_.

Reply:
Strongly prefer we don't move this to a service as we're just creating a lot of boilerplate when a job itself is a service.